ludwig/automl/auto_tune_config.py

```diff
@@ -0,0 +1,164 @@
+import copy
+from collections import OrderedDict
+
+import psutil
+import ray
+
+try:
+    import GPUtil
```

Comment:
Can we add this to the `requirements_ray.txt` instead of asking the user to install this by hand?

Reply:
We should probably move the `import ray` in here, too, given the error message.